cargo-apk/src/apk.rs

```diff
@@ -47,49 +47,64 @@ impl<'a> ApkBuilder<'a> {
             Artifact::Root(name) => format!("rust.{}", name.replace("-", "_")),
             Artifact::Example(name) => format!("rust.example.{}", name.replace("-", "_")),
         };
-        let package_label = self
-            .manifest
-            .metadata
-            .apk_label
-            .as_deref()
-            .unwrap_or(artifact.name())
-            .to_string();
 
-        let config = Config {
+        // Set default Android manifest values
+        let mut manifest = self.manifest.android_manifest.clone();
+        manifest.package = package_name;
+        manifest.version_name = Some(self.manifest.version.clone());
+        manifest.version_code = Some(VersionCode::from_semver(&self.manifest.version)?.to_code(1));
+
+        manifest.sdk.target_sdk_version = manifest
+            .sdk
+            .target_sdk_version
+            .or(Some(self.ndk.default_platform()));
```

Comment:
Could possibly use `get_or_insert(_with)` here (and below), even though you don't need to _get_ the item (dunno if std already has a function for that usecase):
```suggestion
        manifest
            .sdk
            .target_sdk_version
            .get_or_insert(self.ndk.default_platform());
```